src/pint/scripts/event_optimize.py

```diff
@@ -151,7 +151,7 @@ def profile_likelihood(phs, *otherargs):
     Likelihood is calculated as per eqn 2 in Pletsch & Clark 2015.
     """
     xvals, phases, template, weights = otherargs
-    phss = phases.astype(np.float64) + phs
+    phss = phases.astype(np.float64) + phs.astype(np.float64)
```

Comment:
Another instance where data are cast explicitly to 64-bit.  Here it seemed like `phases` was already 64-bit, and then `phs` was added to that, so there would be no/minimal loss in precision.

Reply:
@kerrm : do you think this is OK?  Probably it was silently casting without telling us, so maybe we just have to do it explicitly now.

Reply:
As a general approach, `astype(float)` might be considered instead of using the Numpy type. Python uses 64-bit floats anyway in most architectures, and `float` is sure to be supported over time (while, e.g., Numpy  has recently deprecated and eliminated `np.float`)

Reply:
Thanks, @matteobachetti .  I think it might be better to cast to an explicit precision here rather than leave it architecture dependent.  That way if the architecture doesn't support it, it will fail explicitly rather than just work with altered precision.  But maybe I'm thinking about this the wrong way.